ansible/roles/helios_broker_setup/tasks/create_secrets.yml

```diff
@@ -0,0 +1,93 @@
+---
+
+  - name: Install/update 'apb tools' package if the '{{ apb_tools_cmd }}' command does NOT exist (RedHat OS Env)
```

Comment:
Change to:  " Install/update 'apb tools' package on (RedHat OS Env)"

Reply:
will update